src/components/modals/HelpModal.tsx

```
@@ -0,0 +1,666 @@
+                  <p className="text-gray-600 mb-4">This video tutorial is coming soon!</p>
+                  <Button variant="outline">
+                    <MessageCircle className="w-4 h-4 mr-2" />
+                    Request This Tutorial
+                  </Button>
+                </div>
+              ) : (
+                <div className="bg-gray-50 rounded-lg p-6">
+                  <div
+                    className="text-gray-800 leading-relaxed"
+                    dangerouslySetInnerHTML={{
+                      __html: currentItem.content?.replace(/\n/g, '<br>').replace(/#{1,6}\s/g, '<strong>').replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>') || ''
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)